### DISCUSSION TITLE
请求aarch64的植物大战僵尸

### DISCUSSION THREAD

**Discussion by N00k0428** [Gamuxorg/bbs#416] [投递和请求游戏]
请求aarch64的植物大战僵尸，或者其他aarch64/arm64无限关卡的小游戏。感谢

**Comment by N00k0428** [comment]
不需要安装steam的

**Comment by kero990** [comment]
https://bbs.deepin.org/zh/post/272294

**Reply by N00k0428** [reply]
没网没微信/(ㄒoㄒ)/~~

**Reply by kero990** [reply]
> 没网没微信/(ㄒoㄒ)/~~

新的移植版来了
https://www.linuxgame.cn/4509-2
作者是通过逆向工程，完整重写了年度版的引擎，现在基本是原汁原味
